network/kademlia_test.go

```diff
@@ -370,80 +390,110 @@ func binStr(a *BzzAddr) string {
 	return pot.ToBin(a.Address())[:8]
 }
 
-func TestSuggestPeerFindPeers(t *testing.T) {
-	tk := newTestKademlia(t, "00000000")
-	tk.On("00100000")
-	tk.checkSuggestPeer("<nil>", 0, false)
+//Tests peer suggestion. Depends on expectedMinBinSize implementation
```

Comment:
I find this test has a rather long and winding narrative. Please consider splitting it up into themes, eg:

* test for peer suggest sequence
* test for saturation change
* test for depth change
* test for not callable
* ...?

Reply:
Yeah, I agree. Will try to split it in different tests